openhtmltopdf-pdfbox/src/main/java/com/openhtmltopdf/pdfboxout/PdfRendererBuilder.java

```diff
@@ -153,171 +150,180 @@ else if (font.pdfontSupplier != null) {
-		return this.useFont(supplier, fontFamily, 400, FontStyle.NORMAL, true);
-	}
-
-	/**
-	 * Set a producer on the output document
-	 *
-	 * @param producer
-	 *            the name of the producer to set defaults to openhtmltopdf.com
-	 * @return this for method chaining
-	 */
-	public PdfRendererBuilder withProducer(String producer) {
-		state._producer = producer;
-		return this;
-	}
-
-	/**
-	 * List of caches available.
-	 */
-	public enum CacheStore {
-	    
-	    /**
-	     * Caches font metrics, based on a combined key of family name, weight and style.
-	     * Using this cache avoids loading fallback fonts if the metrics are already in the cache
-	     * and the previous fonts contain the needed characters.
-	     */
-	    PDF_FONT_METRICS;
-	}
-	
-	/**
-	 * Use a specific cache. Cache values should be thread safe, so provided your cache store itself
-	 * is thread safe can be used accross threads.
-	 * @return this for method chaining.
-	 * @see CacheStore
-	 */
-	public PdfRendererBuilder useCacheStore(CacheStore which, FSCacheEx<String, FSCacheValue> cache) {
-	    state._caches.put(which, cache);
-	    return this;
-	}
-
-	/**
-	 * Set a PageSupplier that is called whenever a new page is needed.
-	 * 
-	 * @param pageSupplier 
-	 *            {@link PageSupplier} to use
-	 * @return this for method chaining.
-	 */
-	public PdfRendererBuilder usePageSupplier(PageSupplier pageSupplier) {
-		state._pageSupplier = pageSupplier;
-		return this;
-	}
-
-	/**
-	 * Various level of PDF/A conformance:
-	 *
-	 * PDF/A-1, PDF/A-2 and PDF/A-3
-	 */
-	public enum PdfAConformance {
-		NONE(-1, "", 0f),
-		PDFA_1_A(1, "A", 1.4f), PDFA_1_B(1, "B", 1.4f),
-		PDFA_2_A(2, "A", 1.7f), PDFA_2_B(2, "B", 1.7f), PDFA_2_U(2, "U", 1.7f),
-		PDFA_3_A(3, "A", 1.7f), PDFA_3_B(3, "B", 1.7f), PDFA_3_U(3, "U", 1.7f);
-
-		PdfAConformance(int part, String value, float pdfVersion) {
-			this.part = part;
-			this.value = value;
-			this.pdfVersion = pdfVersion;
-		}
-
-		private final int part;
-		private final String value;
-		private final float pdfVersion;
-		
-		public String getConformanceValue() {
-		    return this.value;
-		}
-
-		public int getPart() {
-			return this.part;
-		}
-
-		public float getPdfVersion() {
-			return this.pdfVersion;
-		}
-	}
+        return this;
+    }
+
+    /**
+     * Whether to conform to PDF/UA or Accessible PDF. False by default.
+     *
+     * @see PdfRendererBuilder#usePdfUaAccessibility
+     * @deprecated Misspelled method name
+     */
+    @SuppressWarnings("SpellCheckingInspection")
+    @Deprecated
+    public PdfRendererBuilder usePdfUaAccessbility(boolean pdfUaAccessibility) {
+        return usePdfUaAccessibility(pdfUaAccessibility);
+    }
+
+    /**
+     * Whether to conform to PDF/UA or Accessible PDF. False by default.
+     */
+    public PdfRendererBuilder usePdfUaAccessibility(boolean pdfUaAccessibility) {
+        this.state._pdfUaConform = pdfUaAccessibility;
+        return this;
+    }
```

Comment:
This is the only real change. I ran the IntelliJ formatter, so the review is a bit confusing. If you want to isolate the change, tell GitHub to ignore commit `a9a5...`